docs/For User/Account keys/02 Multisig.md

```diff
@@ -0,0 +1,256 @@
+First import the public keys of `test3` into your keyring.
+
+```linux
+imversed keys add \
+    test3 \
+    --pubkey=imversedpub1addwnpepqgcxazmq6wgt2j4rdfumsfwla0zfk8e5sws3p3zg5dkm9007hmfysxas0u2
+```
+
+Generate the multisig key with 2/3 threshold.
+
+```linux
+imversed keys add \
+    multi \
+    --multisig=test1,test2,test3 \
+    --multisig-threshold=2
+```
+
+You can see its address and details:
+
+```linux
+imversed keys show multi
+
+- name: multi
+  type: multi
+  address: imversed1e0fx0q9meawrcq7fmma9x60gk35lpr4xk3884m
+  pubkey: imversedpub1ytql0csgqgfzd666axrjzq3mxw59ys6yqcd3ydjvhgs0uzs6kdk5fp4t73gmkl8t6y02yfq7tvfzd666axrjzq3sd69kp5usk492x6nehqjal67ynv0nfqapzrzy3gmdk27la0kjfqfzd666axrjzq6utqt639ka2j3xkncgk65dup06t297ccljmxhvhu3rmk92u3afjuyz9dg9
+  mnemonic: ""
+  threshold: 0
+  pubkeys: []
+
+```
+
+Let's add 10 imversed to the multisig wallet:
+
+```linux
+imversed tx bank send \
+    test1 \
+    imversed1e0fx0q9meawrcq7fmma9x60gk35lpr4xk3884m \
+    10000000000000000000aimv \
+    --chain-id=imversed_9000-4 \
+    --gas=auto \
+    --fees=1000000aimv \
+    --broadcast-mode=block
+```
+
+### Step 2: Create the multisig transaction
+We want to send 5 IMV from our multisig account to `imversed1rgjxswhuxhcrhmyxlval0qa70vxwvqn2e0srft`.
+
+```linux
+imversed tx bank send \
+    imversed1rgjxswhuxhcrhmyxlval0qa70vxwvqn2e0srft \
+    imversed157g6rn6t6k5rl0dl57zha2wx72t633axqyvvwq \
+    5000000000000000000aimversed \
+    --gas=200000 \
+    --fees=1000000aimversed \
+    --chain-id=imversed_9000-4 \
+    --generate-only > unsignedTx.json
+```
+
+The file `unsignedTx.json` contains the unsigned transaction encoded in JSON.
+
+```linux
+{
+  "body": {
+    "messages": [
+      {
+        "@type": "/cosmos.bank.v1beta1.MsgSend",
+        "from_address": "imversed1rgjxswhuxhcrhmyxlval0qa70vxwvqn2e0srft",
+        "to_address": "imversed157g6rn6t6k5rl0dl57zha2wx72t633axqyvvwq",
+        "amount": [
+          {
+            "denom": "aimv",
+            "amount": "5000000000000000000"
+          }
+        ]
+      }
+    ],
+    "memo": "",
+    "timeout_height": "0",
+    "extension_options": [],
+    "non_critical_extension_options": []
+  },
+  "auth_info": {
+    "signer_infos": [],
+    "fee": {
+      "amount": [
+        {
+          "denom": "aimv",
+          "amount": "1000000"
+        }
+      ],
+      "gas_limit": "200000",
+      "payer": "",
+      "granter": ""
+    }
+  },
+  "signatures": []
+}
+```
+
+### Step 3: Sign individually
+
+Sign with `test1` and `test2` and create individual signatures.
+
+```linux
+imversed tx sign \
+    unsignedTx.json \
+    --multisig=imversed1e0fx0q9meawrcq7fmma9x60gk35lpr4xk3884m \
+    --from=test1 \
+    --output-document=test1sig.json \
+    --chain-id=imversed_9000-4
+```
+```linux
+imversed tx sign \
+    unsignedTx.json \
+    --multisig=imversed1e0fx0q9meawrcq7fmma9x60gk35lpr4xk3884m \
+    --from=test2 \
+    --output-document=test2sig.json \
+    --chain-id=imversed_9000-4

```

Comment:
Change **imversed_9000-4** to **imversed_5555555-1**

Reply:
Updated